ChangeLog.d/fix-compilation-with-djgpp.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix:
+  * DJGPP does not provide `suseconds_t`, but defines `__unix__`
+  * `net_sockets.c` uses it for `mbedtls_net_usleep()`
+  * This fix makes use of the alternate codepath w/o `suseconds_t` if `__DJGPP__` is defined
```

Comment:
This is a bit too detailed for a changelog entry: changelog entries are meant for users. Also, the format is plaintext, no `code formatting`. Also, each bullet starts a separate entry: what you've written here is three entries that may be split or reordered, not a single entry. Continuation lines . On the other hand, the changelog entry should reference the issue (https://github.com/Mbed-TLS/mbedtls/pull/9810). Suggestion:

```
Bugfix
   * Fix compilation on MS-DOS DJGPP. Fixes #9810.
```

Reply:
Thanks for the review and the heads up.

I changed the changelog file in all PRs according to your suggestion.